contracts/src/utils/protocol.ts

```diff
@@ -16,23 +16,24 @@ const deployMock = async(hre: HardhatRuntimeEnvironment, name: string): Promise<
     return result.address
 }
 
-export const getProtocolManagerAddress = async(hre: HardhatRuntimeEnvironment): Promise<string> => {
+export const getProtocolManagerAddress = async(hre: HardhatRuntimeEnvironment, owner: AddressLike, registry: AddressLike): Promise<string> => {
     const chainId = await hre.getChainId()
 
     // For the tests we deploy a mock for the manager
-    if (chainId === "31337") return deployMock(hre, "ManagerMock")
+    if (chainId === "31337") return await(await(await hre.ethers.getContractFactory("TestSafeProtocolManager")).deploy(owner, registry)).getAddress();
```

Comment:
Why do we change this? Tests should not rely on a specific implementation. Even if we change this then we need to adjust the comment above. Also the method was meant as a `get` it should not require any arguments as it should return a deployed instance.